pymodbus/server/simulator/http_server.py

```diff
@@ -118,7 +118,7 @@ def __init__(
         self,
         modbus_server: str = "server",
         modbus_device: str = "device",
-        http_host: str = "0.0.0.0",
+        http_host: str = "localhost",
```

Comment:
That is wrong, local host is 127.0.0.1 whereas 0.0.0.0 allows bind on all interfaces.

Reply:
Please add The B code, 0.0.0.0 is the standard for listen.